src/pymovements/dataset/dataset.py

```diff
@@ -772,7 +774,13 @@ def compute_event_properties(
             for column in self.fileinfo['gaze'].columns
             if column != 'filepath'
         ]
-
+        event_property_names = [property[0] for property in processor.event_properties]
+        existing_columns = set(self.events[0].columns) & set(event_property_names)
+        if existing_columns:
+            raise ValueError(
```

Comment:
This is not covered by any tests anymore. Please add a test here:

https://github.com/aeye-lab/pymovements/blob/6245bbfceacd018a6f880a4b8f0f3f44ba2acacd/tests/unit/dataset/dataset_test.py#L1603-L1619

Reply:
Hi @xiaotiansu maybe you overlooked this, there's one final test needed to get full coverage.

After adding the test we'll be ready to merge. Thanks for all the work so far!